org.eclim.jdt/test/junit/org/eclim/plugin/jdt/command/doc/GetElementDocCommandTest.java

```diff
@@ -168,4 +169,14 @@ public void executeJdkElements()
         "|Object[0]| |java[1]|.|util[2]|.|Map[3]|.put(|Object[4]| key, |Object[5]| value)\n\n" +
         "Associates the specified value with the specified key in this map"));
   }
+
+  @Test
+  public void htmlFlag(){
+    String results = (String)
+      Eclim.execute(new String[]{
+        "java_element_doc", "-p", Jdt.TEST_PROJECT,
+        "-f", TEST_FILE, "-o", "255", "-l", "11", "-e", "utf-8", "-h", "true"
+      });
+    Assert.assertTrue("HTML Format expected", results.startsWith("<html><head>"));
```

Comment:
Can you stick with the existing coding style and omit the `Assert.` prefix here.